kubernetes/kubernetes.cabal

```diff
@@ -44,7 +44,7 @@ library
     , http-client >=0.5 && <0.6
     , http-client-tls
     , http-media >= 0.4 && < 0.8
-    , http-types >=0.8 && <0.12
+    , http-types >=0.8 && <0.13
```

Comment:
This file is auto-generated by swagger-codegen. To make a change, we need to modify the source at https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/haskell-http-client/haskell-http-client.cabal.mustache.